packages/commonwealth/client/scripts/views/pages/CommunityManagement/StakeIntegration/StakeIntegration.tsx

```diff
@@ -21,19 +29,41 @@ const StakeIntegration = () => {
     return <PageNotFound />;
   }
 
+  const community = app.chain.meta;
+  const communityChainId = `${
+    community.ChainNode?.ethChainId || community.ChainNode?.cosmosChainId
+  }`;
+  const selectedAddress = app.user.addresses.find(
```

Comment:
I think you can use this `useUserActiveAccount`

Reply:
we actually need an instance of `AddressInfo` here, the `userUserActiveAccount` returns an `Account` instance